tests/test_utils.py

```diff
@@ -3,4 +3,23 @@
 """
 import os
 
+from networkx import Graph
+
+from agh_graphs.utils import gen_name
+
 visualize_tests = 'VISUALIZE_TESTS' in os.environ and os.environ['VISUALIZE_TESTS'] == 'true'
+
+
+def addTriangle(graph: Graph, node, attr):
```

Comment:
Hmmm... mam trudności w zrozumieniu co ta funkcja robi i jak jej użyć. Z nazwy wynika, że dodaje trójkąt, a tak naprawdę dodaje trzy wierzchołki E, które łączy z jakimś istniejącym. Ponadto nie wiem skąd akurat takie współrzędne i dlaczego osobno przekazujemy atrybuty, a osobno ID wierzchołka. Te atrybuty nie są atrybutami wierzchołka `node`? A jeśli nie są to czemu podajemy je w formie mapy a nie `x`, `y`, `layer`? Dodatkowo nazwa jest camel casem a nie snake casem i brakuje końca linii na końcu pliku.

PS. Robię review tego bo ta funkcja została dodana do `test_utils.py`, a więc jest dla innych do użytku.

Reply:
Ta funkcja była używana w testach do P6, po czym myślałem ze moze też się nam przydać więc przeniosłem ją do test_utils. Ale w tym momencie my na pewno jej juz nie uzywamy

Reply:
To w sumie możesz wycofać zmianę, która przenosi tą funkcję tutaj

Reply:
wycofana